feat: support aggregate search

PainterPuppets opened the pull request:
reslove: https://github.com/Magickbase/ckb-explorer-public-issues/issues/519

PainterPuppets commented on src/services/ExplorerService/fetcher.ts:
```diff
@@ -293,6 +314,14 @@ export const apiFetcher = {
+    v1Get<AggregateSearchResult[]>('suggest_queries', {
+      params: {
+        q: param,
+        filter_by: 0,
```
The api provided by the backend, the code is here: https://github.com/nervosnetwork/ckb-explorer/pull/1790/files#diff-240d8fa62d7d98e879be4a2dc4a31b67265baee47f7f419a67226a483c71ebd7R10

Keith-CY replied:
It can be defined as an enum as
```
0 for aggregate_query
1 for single_query
```
to make it clear

PainterPuppets commented:
supported token collection result display

<img width="1115" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/22258327/29a9de62-d4a4-4ebd-bed0-4d25094b5039">

FrederLu commented:
PRD  https://vs0cjf.axshare.com/?id=908ssd&g=1&p=%E6%90%9C%E7%B4%A2%E4%BC%98%E5%8C%96%E8%AF%B4%E6%98%8E&sc=3

1、When the number of blocks being searched is greater than the current number of blocks, clicking Enter will jump to the block countdown page.
<img width="490" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/38eecb3b-56f5-4d45-9058-90498a7a3e64">
<img width="756" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/bf776b56-18b9-4e8c-9afe-24d9a71fdf39">


---
2、When searching at the top, Enter or clicking the search result will not jump to the corresponding page.
<img width="402" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/36f8b0fc-81b6-4646-aae5-59fa52babe98">

---
3、`Lock script`(0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8) and `Type script`(0x8536c9d5d908bd89fc70099e4284870708b6632356aad98734fcf43f6f71c304)  is not Chinese when the page displays Chinese.
<img width="1303" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/9ae52b61-c183-4ce1-97b3-d623ec2a2462">

<img width="1083" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/f1a94a63-3ebd-4a04-b0d7-73bdbf831c3e">
<img width="846" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/a1fe43c5-10f9-48ba-8f1b-d90a85741476">
When using type script at the same time, the search results display `text`?

---
4、Use btc address but no search results.
Btc address: tb1qwq82nch2wc0jtymavfv2cxsju9trghn0ykhqx8
https://ckb-explorer-frontend-in-magickbase-repo-8dt95odvy-magickbase.vercel.app/address/tb1qwq82nch2wc0jtymavfv2cxsju9trghn0ykhqx8
<img width="1318" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/a40891f3-db26-4ce3-a623-84f991f2ee27">

---
5、Use btc tx but no search results.
BTC Tx :4ea970151577aff9f134bd50ce666d304c979f8d204c4fa8f4cd0a1bf7c3bbd1
https://ckb-explorer-frontend-in-magickbase-repo-8dt95odvy-magickbase.vercel.app/transaction/0x769c208860f364c602afb89c11887c7d8a8630bebeeb4d6ac2c65b6a3362d760
<img width="1307" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/92fe85fa-e67a-4ad1-be98-f9248ffc985f">

Keith-CY commented:
Any update on this feature @PainterPuppets

PainterPuppets commented:
supported search result category filter & did / btc address search
<img width="522" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/22258327/7fbce6f4-8fa6-4d52-af10-d561ab2701b3">
<img width="453" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/22258327/792ee887-a345-4a31-87ca-5c0ef311bee4">

PainterPuppets commented:
> PRD https://vs0cjf.axshare.com/?id=908ssd&g=1&p=%E6%90%9C%E7%B4%A2%E4%BC%98%E5%8C%96%E8%AF%B4%E6%98%8E&sc=3
> 
> UI https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=35067-23245&mode=design&t=f9reojugIfk2nzmp-0
> 
> 1、When the number of blocks being searched is greater than the current number of blocks, clicking Enter will jump to the block countdown page. <img alt="image" width="490" src="https://private-user-images.githubusercontent.com/25676576/329914961-38eecb3b-56f5-4d45-9058-90498a7a3e64.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTQ5NjEtMzhlZWNiM2ItNTZmNS00ZDQ1LTkwNTgtOTA0OThhN2EzZTY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjYTc3YmNmNzU2MGZjOTNkNDg2NTdmNTM3NjM3MWRlNWU0MDZkOGFmNGUwMDFjMjYyMzY4MDJmNmVhZDQ1OTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1oHXGEFjzEEALh-Y6neJwYO84O6cHo05OtPL9vLybik"> <img alt="image" width="756" src="https://private-user-images.githubusercontent.com/25676576/329915372-bf776b56-18b9-4e8c-9afe-24d9a71fdf39.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTUzNzItYmY3NzZiNTYtMThiOS00ZThjLTlhZmUtMjRkOWE3MWZkZjM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjNjg0ODFmM2Q4MDgzNGJlMmI0YzUwMTY1N2UxNzk0MGJiOTU1NGIxYjVlNWExMzRlMDBlMzFhOTY1Mzk1OWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FVeT7cvYzydOzfGYANqsaqAb0szNFEtCTn1el2i6JbQ">
> 
> 2、When searching at the top, Enter or clicking the search result will not jump to the corresponding page. <img alt="image" width="402" src="https://private-user-images.githubusercontent.com/25676576/329915747-36f8b0fc-81b6-4646-aae5-59fa52babe98.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTU3NDctMzZmOGIwZmMtODFiNi00NjQ2LWFhZTUtNTlmYTUyYmFiZTk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRkNGQ1ODJkZjA4ZmU0ZjJkN2FiNGY1YmY2OTYwOWVlYjA0MzI0MWYzMmQzNGM1NzBlOGE0YWZhNWU0NmM0ZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Bo9avp23g9OVrHW8-LfJHVBZKHxQv11oK1V6crZ7X98">
> 
> 3、`Lock script`(0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8) and `Type script`(0x8536c9d5d908bd89fc70099e4284870708b6632356aad98734fcf43f6f71c304) is not Chinese when the page displays Chinese. <img alt="image" width="1303" src="https://private-user-images.githubusercontent.com/25676576/329916417-9ae52b61-c183-4ce1-97b3-d623ec2a2462.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTY0MTctOWFlNTJiNjEtYzE4My00Y2UxLTk3YjMtZDYyM2VjMmEyNDYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4ZDM0YjkxNDkwMjFiOWFjODA5MmY0ZDdhNzdjY2VjZjE2MGQyYTBhMDQxZDY5YzM2NTMxYzY5ZTU0NzlhZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.J2ruX_oyF9B9mPhhgABF_p_gMvX_RKFqXvqx8-Y20QY">
> 
> <img alt="image" width="1083" src="https://private-user-images.githubusercontent.com/25676576/329916315-f1a94a63-3ebd-4a04-b0d7-73bdbf831c3e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTYzMTUtZjFhOTRhNjMtM2ViZC00YTA0LWIwZDctNzNiZGJmODMxYzNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZkMWNjYzUzM2U0MmQ2ZDA0ZDhhOGEwZGQ4NDc4MjNhYTA0N2JmZmIwMWY4NGM5M2QxZWYzNjJkNjRjOWZjY2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MoGf9t4PitoqnLKnWlOBgnzrV4yiEKpi6UcYVUOKwVY"> <img alt="image" width="846" src="https://private-user-images.githubusercontent.com/25676576/329916856-a1fe43c5-10f9-48ba-8f1b-d90a85741476.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTY4NTYtYTFmZTQzYzUtMTBmOS00OGJhLThmMWItZDkwYTg1NzQxNDc2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc5YzcyMzM5YWZhNjRmM2NlOWNhOGJhNWM1ZmQ3N2U5MjQ2YjEzOGVlOGQ3Mjg0MzBjMTM4YzMzMmYzYzIyOTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.i6-b-VfOCa5LrXBSw-bNYa4M2cmkEwFrE2rdZU4CrBI"> When using type script at the same time, the search results display `text`?
> 4、Use btc address but no search results. Btc address: tb1qwq82nch2wc0jtymavfv2cxsju9trghn0ykhqx8 https://ckb-explorer-frontend-in-magickbase-repo-8dt95odvy-magickbase.vercel.app/address/tb1qwq82nch2wc0jtymavfv2cxsju9trghn0ykhqx8 <img alt="image" width="1318" src="https://private-user-images.githubusercontent.com/25676576/329919134-a40891f3-db26-4ce3-a623-84f991f2ee27.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTkxMzQtYTQwODkxZjMtZGIyNi00Y2UzLWE2MjMtODRmOTkxZjJlZTI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJiOWY3NzEyY2U1OGI1M2U4ZjkyYTI5ODkyYTM1OTdhOWUzMTk5ZTFiOGMyZTFhYTQxY2RhMTgzOTlkYjhmNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ogwoJPEV9NoJYt-1E80St-7ibkRKKmSBxdqBI-jK3Jg">
> 
> 5、Use btc tx but no search results. BTC Tx :4ea970151577aff9f134bd50ce666d304c979f8d204c4fa8f4cd0a1bf7c3bbd1 https://ckb-explorer-frontend-in-magickbase-repo-8dt95odvy-magickbase.vercel.app/transaction/0x769c208860f364c602afb89c11887c7d8a8630bebeeb4d6ac2c65b6a3362d760 <img alt="image" width="1307" src="https://private-user-images.githubusercontent.com/25676576/329919881-92fe85fa-e67a-4ad1-be98-f9248ffc985f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MTk4ODEtOTJmZTg1ZmEtZTY3YS00YWQxLWJlOTgtZjkyNDhmZmM5ODVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3MWY3ZmY3NmFkZTEwNzkwNTQwN2E2ZDU4NDdkMGZjMDJkNGNhYjBiODdkZWUzYzAyZjI1MDFkN2M5M2Y4YmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0d10_y31OUUuaebuqZuLnmAW5yAkyfJOqPtiNSx1pFk">
> 
> 6、Use .bit but no search results. https://pudge.explorer.nervos.org/address/ckt1qzfhdsa4syv599s2s3nfrctwga70g0tu07n9gpnun9ydlngf5vsnwqg9k22096uxuljzg6kjsdcmacxtm2m4vw8yqkeffuhtsmn7gfr262phr0hqe0dtw43cushnmkvy <img alt="image" width="1277" src="https://private-user-images.githubusercontent.com/25676576/330002518-9fccc12d-37d1-472b-bdd3-5be8ae7c2109.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMzAwMDI1MTgtOWZjY2MxMmQtMzdkMS00NzJiLWJkZDMtNWJlOGFlN2MyMTA5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiZGZlZTM4ZGY3ZjJiODhmMzFlNzg1ZmZmN2Y2NmQ4ZmY1ZGFjMjZlYjVlN2Y0MzQ3ZmQ3ODYzM2IxZDk5NGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HvIod6KuAz3DfqYhXlpakysxP-p4ZVDdIT5EVqEI1Fo">
> 
> 7、The data style in the search results is abnormal. Refer to the visible part of data before and after the UI style, using `...` in the middle.
> 
> UI <img alt="image" width="981" src="https://private-user-images.githubusercontent.com/25676576/330006026-2e8a1c80-8346-44c6-a9c9-8c8528dd827d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMzAwMDYwMjYtMmU4YTFjODAtODM0Ni00NGM2LWE5YzktOGM4NTI4ZGQ4MjdkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxZmZkMjc2YzYyMzRiZjg5YzZhOGM2NWZjNGFkYmU5MGViNzRlMDc1MGI2N2U5ZGI1NDk0OTZmMjJiZTEwMmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2XYI__2cknx5JZAdHkuTfFaJd8xPUN-BoH1MV_E31iY"> <img alt="image" width="1204" src="https://private-user-images.githubusercontent.com/25676576/330006344-041d91d8-8e7f-4729-be1f-5dd474e30dd0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMzAwMDYzNDQtMDQxZDkxZDgtOGU3Zi00NzI5LWJlMWYtNWRkNDc0ZTMwZGQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4NDgwOWEwM2RmYzEwMWYwOWNjZTlmMDg2ZTM4YjkzMDlhMDRjZTQ0ZmRhZDkwMGM3N2UyYmM5MDhmYjhjMGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZjpjmH6eiNUAZfhkM8c3JNYq8HLGTZTtn03bJVaYxgo"> <img alt="image" width="312" src="https://private-user-images.githubusercontent.com/25676576/330006613-e7c2e809-7e47-4366-afe4-335a2f0a0dfe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMzAwMDY2MTMtZTdjMmU4MDktN2U0Ny00MzY2LWFmZTQtMzM1YTJmMGEwZGZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYxNzBhNmQ2YTRmNmU1NGEzYzQ4NTRiMTc3OGQ3YjEwNjMzYjUwMmQ0OTE3YWEyY2RjYWE5OTc3MWUyOGI0MjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pz_7XFAfgMmRUhBIBi7mSU768G344qgSJbxWUL6Dx20"> <img alt="image" width="648" src="https://private-user-images.githubusercontent.com/25676576/330007682-597f8548-3faa-4a9b-9760-26e1447ea1f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMzAwMDc2ODItNTk3Zjg1NDgtM2ZhYS00YTliLTk3NjAtMjZlMTQ0N2VhMWYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjZWNlYTI4ZDQ0YmU1NTkxYWViMjAxOTE1ZjIzMzc4MWRlZjE2ODJjZjE1Mzk4MDg3MDc0NzM0MzdlNmZjNWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.PiVf8H-3UiM-mfsCdEiSjxucr-RtcEDYqyiv_Foo40c">
> 
> 8、Like typing `aaaa`, the page lacks multiple data tabs, and the search box lacks filtering options.
> 
> <img alt="image" width="1203" src="https://private-user-images.githubusercontent.com/25676576/329921774-d1e7d402-73cc-4128-8ab3-33d8f1251205.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MjE3NzQtZDFlN2Q0MDItNzNjYy00MTI4LThhYjMtMzNkOGYxMjUxMjA1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI3NDBiNmNhYjdmOWVjZDViNzJkN2QzMGIxOWE3YmI4YTRhZTkwNWNkZTk1ODQxNDI5ZTE5YzRmMzc2NDZhM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hU7TNR99LliREXBFnTUwctmO5vqFjPS8dqbFKYwUCrI">
> UI: <img alt="image" width="926" src="https://private-user-images.githubusercontent.com/25676576/329922364-c33ffa8c-95dd-4732-bc58-b09361accf13.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MjIzNjQtYzMzZmZhOGMtOTVkZC00NzMyLWJjNTgtYjA5MzYxYWNjZjEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI0ZTQwZjA3OGZkYjgzNGFiMWFlNWZhM2YxMTQzZjczMmE4ODFiYTc2YzQ1YTIxYjEzOTM3MzA0ZmVlZjU3MzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7k_uJmonF4lE8DuOpihvn0RTy1Zp6oRt6I0lB8ZZCbI">
> 
> <img alt="image" width="1255" src="https://private-user-images.githubusercontent.com/25676576/329922162-67d8ff25-40da-42f9-a568-9b06b32444e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTY5OTEsIm5iZiI6MTcxNTU5NjY5MSwicGF0aCI6Ii8yNTY3NjU3Ni8zMjk5MjIxNjItNjdkOGZmMjUtNDBkYS00MmY5LWE1NjgtOWIwNmIzMjQ0NGUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEwMzgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2Nzg2NTFhYTUzOTU5NmZmNTgzZWI4YmZlZjQ1ZjFlZGZhNjA3ODVmM2YxNjY5MzA2MzhhMmM0YjAxNDVjYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.L37JYVGjaazikUi-AYqTkhxPvoDK61bly1C4vz-h53E">

1. We don't seem to have a block countdown page at the moment, so i talked to the @Sven-TBD , then PM said we'll ignore the logic for now.
2. I cant reproducing it. do you have a video?
3. fixed
4. supported
5. There doesn't seem to be a way for the front-end to know if a btc transaction is rgb++ or not, and there doesn't seem to be a way to correspond to a ckb transaction, so it may require back-end support for searching btc transactions @rabbitz 
6. supported
7. supported
8. Added filter for search result panel, I felt that the filtering in front of the search box was a bit of a duplicate of the filtering on the search results panel, so I discussed it with the @Sven-TBD and concluded that it could be left off for now

FrederLu commented:
>2. I cant reproducing it. do you have a video?

The operation jumped after this submission.

>5. There doesn't seem to be a way for the front-end to know if a btc transaction is rgb++ or not, and there doesn't seem to be a way to correspond to a ckb transaction, so it may require back-end support for searching btc transactions @rabbitz

 @rabbitz You need to take a look at this.

---
9、The search result style of Type script(0x8536c9d5d908bd89fc70099e4284870708b6632356aad98734fcf43f6f71c304) needs to be the same as Lock script(0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a 8637b17723bbda3cce8) Be consistent.

<img width="1254" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/e187a9c6-7026-4fc6-a1e4-a33b38de09e4">
<img width="1279" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/4d9268bc-23e2-40f8-8de2-cbc06bd62b75">

---
10、The spacing between selections needs to be adjusted according to the UI, and is currently too small.At the same time, the selected state here only requires a single selection. Multiple selections are now possible.
<img width="1279" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/e8ac7880-f611-42f1-81f8-d1f84dbfa170">

rabbitz commented:
![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/990650/9f53ac9f-8a71-4484-b759-408182a52515)

Need to add `0x` in front of btc txid @PainterPuppets

PainterPuppets commented:
> 0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a 8637b17723bbda3cce8

fixed

PainterPuppets commented:
> > 2. I cant reproducing it. do you have a video?
> 
> The operation jumped after this submission.
> 
> > 5. There doesn't seem to be a way for the front-end to know if a btc transaction is rgb++ or not, and there doesn't seem to be a way to correspond to a ckb transaction, so it may require back-end support for searching btc transactions @rabbitz
> 
> @rabbitz You need to take a look at this.
> 
> 9、The search result style of Type script(0x8536c9d5d908bd89fc70099e4284870708b6632356aad98734fcf43f6f71c304) needs to be the same as Lock script(0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a 8637b17723bbda3cce8) Be consistent.
> 
> <img alt="image" width="1254" src="https://private-user-images.githubusercontent.com/25676576/331035017-e187a9c6-7026-4fc6-a1e4-a33b38de09e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwMzUwMTctZTE4N2E5YzYtNzAyNi00ZmM2LWExZTQtYTMzYjM4ZGUwOWU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY0OWFlZGQ1NjViNzllOWI1NGYzNzZkYWVlYjY1YzM5Y2Q3YzdjMGE4OTcyZGY3M2FiN2VkODc4ZDRlZGJlYzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.yIog07hQdNwaiFzVHOFL5NKYqS2HDk627swykgOBHSI"> <img alt="image" width="1279" src="https://private-user-images.githubusercontent.com/25676576/331035180-4d9268bc-23e2-40f8-8de2-cbc06bd62b75.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwMzUxODAtNGQ5MjY4YmMtMjNlMi00MGY4LThkZTItY2JjMDZiZDYyYjc1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM1NThlMTVlN2E5MTcwMWZmMWM1NTA2MGY4ODAxMjgyMTBjOTdkNTBmNGJmMzZjN2RmYTVkZWJhYjkwODM0ZmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kzG2H1AbhlIpqSN6UwjHnYNBwNUBZ4-WCzZ56ByAYfQ">
> 10、The spacing between selections needs to be adjusted according to the UI, and is currently too small.At the same time, the selected state here only requires a single selection. Multiple selections are now possible. <img alt="image" width="1279" src="https://private-user-images.githubusercontent.com/25676576/331035775-e8ac7880-f611-42f1-81f8-d1f84dbfa170.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwMzU3NzUtZThhYzc4ODAtZjYxMS00MmYxLTgxZjgtZDFmODRkYmZhMTcwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg1YzEzZDYzMzYxNjU5MWY3YTE4Nzk5OTAyNDk2MGEwNTYzNTNlNzc0YWEzZjNjMjdlZWMxZWQ3ZjM2MmNiY2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YWURm7FOOEr8-vBekutU482SwOgYQ_RP1lRQNqVqQqE">
> 
> 11、Entering `aa` according to the search logic requires searching for `Token Collection` data. <img alt="image" width="775" src="https://private-user-images.githubusercontent.com/25676576/331037234-d7817080-d106-4d26-a03f-55316b27f2bd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwMzcyMzQtZDc4MTcwODAtZDEwNi00ZDI2LWEwM2YtNTUzMTZiMjdmMmJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlZTQ3NTU4ODM1OWM0ZTQzMGVkMGE1ZmQzYWI5MmNiYTgwYzgxMjI2ZjZmYWRlMzU0NGNiZThmYzVlMjNlMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jUrAaU66srAE8cgzFfPH-jW4TQd-atZdzEBZLpF72TA"> <img alt="image" width="1040" src="https://private-user-images.githubusercontent.com/25676576/331037327-341557af-9d89-431f-a48a-2fe6f4ca61cc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwMzczMjctMzQxNTU3YWYtOWQ4OS00MzFmLWE0OGEtMmZlNmY0Y2E2MWNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4NTA5YmU3NDVkMGM3ODUwYmI0NzcwYmNkOWJiOTU2MjlkMWY5ZTk2ZTcxODFlM2RmN2I0OTExYmRmOTQ4OTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.VMYXxBL2Y6Tmkiql2yXx1ZssuoL1jmMUZhTvj554WsM"> At the same time, the data `aaaaa` should also be displayed when `aa` or `aaa` is entered. <img alt="image" width="1133" src="https://private-user-images.githubusercontent.com/25676576/331038081-66ac9ca3-096a-4799-b9eb-9d1a89b5202c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwMzgwODEtNjZhYzljYTMtMDk2YS00Nzk5LWI5ZWItOWQxYTg5YjUyMDJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzODJhZDJhZWQ0MDQ0ODJjZTFhYmM2NmRkYzYyMWZiN2YwMTUyOTU3ZjM3YjUxNzYwMjRhNjIzZTYwYzRmOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.b9uMBx4UGYI5H-RoxSh07_V-Ql3c8g9TbpkFgC8QEeg">
> 
> 12、The `Token Collection` data displayed on the page does not match the api. <img alt="image" width="1260" src="https://private-user-images.githubusercontent.com/25676576/331053289-13522771-5abc-4cb6-ab80-f62d2b0b6776.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwNTMyODktMTM1MjI3NzEtNWFiYy00Y2I2LWFiODAtZjYyZDJiMGI2Nzc2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUyNjU3OThkODQ1NjIwYzEzMjU4N2ZiOWYwZmNjZDE3NzhjZTM4YTEyYjk1Mjk1YzJlZTAwOGQxZDNjNzVkY2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7lRG83Byzy7rXxiDhEjzHh_blrwChsOxyrVjosi8tx0"> <img alt="image" width="1282" src="https://private-user-images.githubusercontent.com/25676576/331065824-fc3ad0d5-03af-452c-869d-9a0257c7cd49.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwNjU4MjQtZmMzYWQwZDUtMDNhZi00NTJjLTg2OWQtOWEwMjU3YzdjZDQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdmMmY5YTM0N2NiODE1YzY0MzJkN2VmNWIwMzg5NGM5NjE4Mzg1OTMzMTEwNTc3NTk0MmQ2ZDgyN2Y4ZmU4NWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mgEA6n4AMGEHegM-8UE6gjPt3Hm-mk8kK9zxPwnamCo">
> 
> 13、When searching using `BTC Tx:4ea970151577aff9f134bd50ce666d304c979f8d204c4fa8f4cd0a1bf7c3bbd1`, refer to the UI to improve the search result style. There is also the `RGB++` tag. CKB transaction ID (above) - corresponding api ckb_transaction_hash; BTC TXID (below)--corresponding api transaction_hash; <img alt="image" width="644" src="https://private-user-images.githubusercontent.com/25676576/331089368-35f93f35-4af9-488f-a710-499480b7b43c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwODkzNjgtMzVmOTNmMzUtNGFmOS00ODhmLWE3MTAtNDk5NDgwYjdiNDNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUxNThhYTE0MzY1ZGY0ODA2YjU0Y2UxMTkzMTU4ZmFjYTJlYjY5OTFlMDMzZDI4MjYwMzZkMzFkZGZjYjkwZGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FSMpsHkcFt9H3RXY1aLgJHmffQBKpQK6jucGTcA1TnY"> <img alt="image" width="1331" src="https://private-user-images.githubusercontent.com/25676576/331089212-70c99bdd-2387-4fd6-84f0-364d57340795.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDQ4OTQsIm5iZiI6MTcxNTg0NDU5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzEwODkyMTItNzBjOTliZGQtMjM4Ny00ZmQ2LTg0ZjAtMzY0ZDU3MzQwNzk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA3Mjk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjOTk2OTVmNDNmYWQ2ZjdiNzJiZWQ3NmVhYmY0YmMyZTJhYzVjN2JlZDM3YTM5N2U4YTM5ZDM0NmM0OWI0MWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._E3J-7UAyBqHlBumswJAWqdKde17xzYISssaj0fxStM">

fixed

FrederLu commented:
<img width="1311" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/6c8b9085-cc6d-41e2-9429-afcb4730708c">

Due to adjustments to the `btc address`-`tb1qkk0373ap96786j5532n3a8agxnn657d9j7w206` search API, the current page display is inconsistent with the API data and needs to be adjusted.

PainterPuppets commented:
> tb1qkk0373ap96786j5532n3a8agxnn657d9j7w206

fixed

PainterPuppets commented:
> <img alt="image" width="1311" src="https://private-user-images.githubusercontent.com/25676576/331132047-6c8b9085-cc6d-41e2-9429-afcb4730708c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NTE5NjQsIm5iZiI6MTcxNTg1MTY2NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzExMzIwNDctNmM4YjkwODUtY2M2ZC00MWUyLTk0MjktYWZjYjQ3MzA3MDhjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA5Mjc0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzYTRlYTRhNzI2NWM2MzBlNmE1OGY2NDZkODVmOGEyNzdlZDg4NDcwNzE5Yzc4NzAxNGY5OGQ1NmQwMmQzYzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AdftLvuFVHAlLtiMNDLiJ0_21dFX4PR72BcdTNIaMKY">
> Due to adjustments to the `btc address`-`tb1qkk0373ap96786j5532n3a8agxnn657d9j7w206` search API, the current page display is inconsistent with the API data and needs to be adjusted.

fixed

FrederLu commented:
><img width="656" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/89dcb97c-2071-4e2d-b5e9-2b4374cf6e08">

When multiple options appear, the problem of only single selection still exists.

---
<img width="308" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/c21f8a2b-6a5b-47c0-8dde-625b853d77b6">
<img width="1260" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/e6d3984c-5ab0-404e-b319-c337702e9c7d">
14、The search results show a lack of reasonable adaptation. When using `...`, you need to give priority to the later display positions.

PainterPuppets commented:
<img width="317" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/22258327/d7c1ee74-98ca-4e52-8f7b-2b6bad56e116">

my mistake, I thought this was a multiple checkbox.

had fixed

PainterPuppets commented:
> > <img alt="image" width="656" src="https://private-user-images.githubusercontent.com/25676576/331154845-89dcb97c-2071-4e2d-b5e9-2b4374cf6e08.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NTc1OTQsIm5iZiI6MTcxNTg1NzI5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzExNTQ4NDUtODlkY2I5N2MtMjA3MS00ZTJkLWI1ZTktMmI0Mzc0Y2Y2ZTA4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDExMDEzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM1ZmEwZDRlNzE0YmQxMTliMzM2ZDNiODNmYjc5MzA4ZWZhMmIxMzEwYjQ3ODlhMDNlYTEyNDg5MzYzNmM3MDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.BRih9vUz1WeMT3S58uQIdx7P5ZN_D-KMrCS94qvn-9E">
> 
> When multiple options appear, the problem of only single selection still exists.
> 
> <img alt="image" width="308" src="https://private-user-images.githubusercontent.com/25676576/331153245-c21f8a2b-6a5b-47c0-8dde-625b853d77b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NTc1OTQsIm5iZiI6MTcxNTg1NzI5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzExNTMyNDUtYzIxZjhhMmItNmE1Yi00N2MwLThkZGUtNjI1Yjg1M2Q3N2I2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDExMDEzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlMTRkMmIxYjhlYWQwMzZhZTZhMzdmZmRjNWYwYWYxNTMyODgyM2EyZjIxMzdhOWZmZTJiZjMwYTgwMjFkMWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cHlPH8t84dGP-KtQIjeS2X2AyvI6sJMpGG0Lr4NSjdc"> <img alt="image" width="1260" src="https://private-user-images.githubusercontent.com/25676576/331153766-e6d3984c-5ab0-404e-b319-c337702e9c7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NTc1OTQsIm5iZiI6MTcxNTg1NzI5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzExNTM3NjYtZTZkMzk4NGMtNWFiMC00MDRlLWIzMTktYzMzNzcwMmU5YzdkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDExMDEzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlODJmZmMwMmVjM2RlNWMwMGMwOTNhNjVhZTVkMzAyZmM5ZTg2NDZjMzA3ZjY0ZDk5MjE1NGEyZGY2MTljN2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rh1gT8d09W5ynkjU4w8mXMGJCDnOokq_WSmAuvaxxOM">
> 14、The search results show a lack of reasonable adaptation. When using `...`, you need to give priority to the later display positions.
> 
> Reference style <img alt="image" width="268" src="https://private-user-images.githubusercontent.com/25676576/331158214-37beb997-13f4-477d-aa74-252f868f6302.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NTc1OTQsIm5iZiI6MTcxNTg1NzI5NCwicGF0aCI6Ii8yNTY3NjU3Ni8zMzExNTgyMTQtMzdiZWI5OTctMTNmNC00NzdkLWFhNzQtMjUyZjg2OGY2MzAyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDExMDEzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJhM2E2ZmU5NmUzOGUyOGJhY2RlNGJjOTU1MGVjNzVlYTQ1OTQwMGUwOGY4NmFlMzU0MzcxMDYxNzNhYTk2NzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8djesb1gxdxVf8wNSnFbmaKT29ad9z1FRfRHOQLJ6Zg">

my mistake, I thought this was a multiple checkbox.

had fixed & optimize some search result item ui

<img width="279" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/22258327/1a471eb7-7bc2-4be0-82b2-b208d64043ff">
<img width="280" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/22258327/8c2f7db3-095a-4f7c-b929-86e931d01941">

FrederLu commented:
<img width="1008" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/25676576/3de4b50a-6dde-4c46-a9f0-e4299fe006b7">
To avoid ambiguity, it can be written as block height data.
